pom.xml

```diff
@@ -120,7 +120,8 @@
         <yarn.version>v1.5.1</yarn.version>
 
         <!--Project properties-->
-        <ddf.version>2.26.7</ddf.version>
+
+        <ddf.version>2.26.17-SNAPSHOT</ddf.version>
```

Comment:
I'm assuming this DDF version would get updated on the next DDF 2.26.x release.

Reply:
Yes. I had to make changes there for these to work.